### PR TITLE
Whistles

### PR DESCRIPTION
**Pull request by RamonGilabert**
screen shot 2015-12-07 at 4 48 29 pm

### CODE REVIEW

File: Demo/WhisperDemo/WhisperDemo/ViewController.swift [hunk 1]
```diff
@@ -120,11 +129,20 @@ class ViewController: UIViewController {
   }
 
   func presentNotificationDidPress(button: UIButton) {
-    let announcement = Announcement(title: "Ramon Gilabert", subtitle: "Vadym Markov just commented your post", image: UIImage(named: "avatar"))
+    let announcement = Announcement(title: "Ramon Gilabert",
+
```
Comment (vadymmarkov) [inline Demo/WhisperDemo/WhisperDemo/ViewController.swift]: :scream:

Comment (RamonGilabert) [inline Demo/WhisperDemo/WhisperDemo/ViewController.swift]: :smile:

File: Demo/WhisperDemo/WhisperDemo/ViewController.swift [hunk 2]
```diff
@@ -120,11 +129,20 @@ class ViewController: UIViewController {
...
+    subtitle: "Vadym Markov just commented your post", image: UIImage(named: "avatar"))
 
     Shout(announcement, to: self)
   }
 
+  func statusBarButtonDidPress(button: UIButton) {
+    let murmur = Murmur(title: "This is a small whistle",
```
Comment (vadymmarkov) [inline Demo/WhisperDemo/WhisperDemo/ViewController.swift:144]: :cat2:

**Comment by RamonGilabert** [comment]
The code to display a whistle is the following:

```
let murmur = Murmur(title: "This is a small whistle", backgroundColor: UIColor(red: 0.975, green: 0.975, blue: 0.975, alpha: 1))

Whistle(murmur)
```

It will display it as a window in the current context, it supports landscape and portrait. @zenangst, @vadymmarkov, @attila-at-hyper.

The support for landscape and stuff like that needs to be done for Whisper and Shout, but at the moment we don't need it.

**Comment by RamonGilabert** [comment]
screen shot 2015-12-08 at 2 33 39 pm
screen shot 2015-12-08 at 2 33 34 pm

**Comment by vadymmarkov** [comment]
:scream:

**Comment by RamonGilabert** [comment]
:kissing_heart:

**Pull request merged**